README.md

```diff
@@ -16,16 +16,18 @@
 **Table of Contents**
 
 - [Installation](#installation)
-- [Usage](#usage)
-- [Library features](#library-features)
-    - [New Relic configuration](#new-relic-configuration)
-    - [Rack middleware](#rack-middleware)
-    - [Database configuration](#database-configuration)
-    - [Sidekiq](#sidekiq)
-    - [HireFire Workers](#hirefire-workers)
-    - [Logging](#logging)
-    - [Google Oauth](#google-oauth)
+- [Configuration and usage](#configuration-and-usage)
```

Comment:
Can we keep the "library usage" v "command features" sections? Users have been confused by the dual nature of `roo_on_rails` (it's both a library/framework complement _and_ a CLI, usable separately).